packages/atomic/src/components/common/atomic-tab/atomic-tab.tsx

```diff
@@ -0,0 +1,78 @@
+import {buildTab, Tab, TabState} from '@coveo/headless';
+import {Component, h, Prop, State} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {randomID} from '../../../utils/utils';
+import {Bindings} from '../../search/atomic-search-interface/atomic-search-interface';
+import {Button} from '../button';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-tab',
+  styleUrl: './atomic-tab.pcss',
+  shadow: true,
+})
+export class AtomicTab implements InitializableComponent {
```

Comment:
Unless there was some discussion I missed about it, as far as I'm aware, we **don't** want a `atomic-tab` component. We should make this component IPX-specific and give it a name like `atomic-ipx-tab` (same for `atomic-ipx-tabs` of course).

Reply:
Done ✔️ !